autogluon/task/tabular_prediction/presets_configs.py

```diff
@@ -0,0 +1,68 @@
+
+
+def unpack(g):
+    def _unpack_inner(f):
+        def _call(**kwargs):
+            return f(**g(**kwargs))
+        return _call
+    return _unpack_inner
+
+
+# Dictionary of preset fit() parameter configurations.
+preset_dict = dict(
+    # Best quality with little consideration to inference time or disk usage. Achieve even better results by specifying a large time_limits value.
+    # Recommended for applications that benefit from the best possible model quality.
+    best_quality={'auto_stack': True},
+
+    # Identical to best_quality but additionally trains refit_full models that have slightly lower quality but are over 10x faster during inference and require 10x less disk space.
+    best_quality_with_high_quality_refit={'auto_stack': True, 'refit_full': True},
+
+    # High quality with fast inference. ~10x-200x faster inference and ~10x-200x lower disk usage than `best_quality`.
+    # Recommended for applications that require reasonable inference speed and/or model size.
+    high_quality_fast_inference_only_refit={'auto_stack': True, 'refit_full': True, 'set_best_to_refit_full': True, 'save_bagged_folds': False},
+
+    # Good quality with very fast inference. ~4x faster inference and ~4x lower disk usage than `high_quality_fast_inference_only_refit`.
+    # Recommended for applications that require fast inference speed.
+    good_quality_faster_inference_only_refit={'auto_stack': True, 'refit_full': True, 'set_best_to_refit_full': True, 'save_bagged_folds': False, 'hyperparameters': 'light'},
+
+    # Medium quality with very fast inference and very fast training time. ~20x faster training than `good_quality_faster_inference_only_refit`.
+    # This is the default preset in AutoGluon, but should generally only be used for quick prototyping, as `good_quality_faster_inference_only_refit` results in significantly better quality and faster inference time.
+    medium_quality_fastest_train={'auto_stack': False},
```

Comment:
what about 'refit_full' version of this?

Reply:
what about even faster/worse-accuracy options like:

{'auto_stack': False, 'hyperparameters': 'light'}

Reply:
also do we want to call this `fastest_train`? There are certainly configs that will give faster training that this, so I'd call it `faster_train`

Reply:
> what about 'refit_full' version of this?

I don't believe that the refit_full version of this meaningfully expands the pareto frontier compared to the previous options (good_quality_faster_inference_only_refit would likely be faster in inference and much better in accuracy). The only part it wins in is training time, but this would only matter in very low time_limits scenarios, and in future I plan to add adaptive bagging where it will choose to not bag if there is not enough time, which should generalize this case without needing a config preset. In general I think we should steer people towards bagging + stacking solutions.

Reply:
> what about even faster/worse-accuracy options like:
> 
> {'auto_stack': False, 'hyperparameters': 'light'}

I think that the default is already very fast, and I don't want to encourage any presets that are worse than the default.

Reply:
> also do we want to call this `fastest_train`? There are certainly configs that will give faster training that this, so I'd call it `faster_train`

Renamed to faster_train